app/src/screens/PersonCredential.tsx

```diff
@@ -100,17 +99,13 @@ export default function PersonCredential() {
 
   // Use this function to accept the attestation proof request.
   const acceptAttestationProofRequest = async (agent: BifoldAgent, proofRequest: ProofExchangeRecord) => {
-    // Sanity check to make sure we have the necessary credentials
-    const credential = await getAvailableAttestationCredentials(agent)
```

Comment:
should this be removed from attestation helpers? idk if it's being used anywhere else

Reply:
Yeah there are a couple of changes we should make but I wanted to keep this PR to be only logging changes